manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -372,6 +372,32 @@ status:
   conditions: []
   storedVersions: []
 ---
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
We will need to have a special makefile target for deploying the mutation VWH config while its still in dev, per:

https://docs.google.com/document/d/1MdchNFz9guycX__QMGxpJviPaT_MZs8iXaAFqCvoXYQ/edit#heading=h.m5f18i5popfu

Reply:
For now this will be disabled by the enableMutation flag (`[here](https://github.com/open-policy-agent/gatekeeper/pull/881/files#diff-b51dd886772df2afc559857517988e0b2c597cc944583aa47891f6de293b4f21R65)`)

I will add the setting of the flag to "deploy-mutation" (which we already have [here](https://github.com/open-policy-agent/gatekeeper/blob/f3ec64a56154cd991e98387329083d243c967879/Makefile#L143)) in a separate patch.

Reply:
We should make sure the MutatingWebhookConfiguration doesn't enter the K8s API server at all, as a downed (or never available) webhook can interfere with the availability of the cluster (e.g. timeouts).

Reply:
I modified the manifests and makefile to only include the Mutating webhook in the deploy-mutation target.